wordfreq/tokens.py

```diff
@@ -0,0 +1,113 @@
+import regex
+import unicodedata
+
+
+TOKEN_RE = regex.compile(r"""
+    # Case 1: a special case for Chinese and Japanese
+    # -----------------------------------------------
+
+    # When we see characters that are Han ideographs (\p{IsIdeo}) or hiragana
+    # \p{Script=Hiragana}, we allow a sequence of those characters to be glued
```

Comment:
(while I'm making stupid typographical comments) This line is inconsistent with the previous one about parentheses.